test/sanity/issue6136-nwjs-history-escapes-iframe/test.py

```diff
@@ -0,0 +1,56 @@
+import time
+import os
+import platform
+import selenium
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+
+chrome_options = Options()
+testdir = os.path.dirname(os.path.abspath(__file__))
+chrome_options.add_argument('nwapp=' + testdir)
+
+driver = webdriver.Chrome(executable_path=os.environ['CHROMEDRIVER'], chrome_options=chrome_options)
+driver.implicitly_wait(2)
+time.sleep(2)
+try:
+    print driver.current_url
+
+    print 'Click Next button'
+    driver.find_element_by_tag_name('a').click()
+    time.sleep(1)
```

Comment:
Please remove time.sleep() as I reviewed in other cases.

Reply:
Replaced it using `wait_for_element_tag` which exist in file `nw_util.py`, and this PR depends on #6397